scripts/src/mock_client.ts

```diff
@@ -378,8 +389,8 @@ async function testRegisterNewUser() {
 
 async function main() {
 	await earnFlow();
-	await didNotApproveTOS();
-	await testRegisterNewUser();
+	// await didNotApproveTOS();
```

Comment:
this on purpose or for local testing?

Reply:
purpose because there is no approval now